components/debug/debug-dialog.tsx

```
@@ -1,83 +1,153 @@
-      <DialogContent className="max-h-[100svh] max-w-full overflow-y-auto sm:max-h-[90svh] sm:max-w-6xl">
+      <DialogContent className="max-h-[100svh] max-w-full overflow-y-auto sm:max-h-[90svh] sm:max-w-[82vw]">
         <DialogHeader>
-          <DialogTitle>Debug Bot Logs</DialogTitle>
-          <DialogDescription>Bot ID: {bot_uuid}</DialogDescription>
+          <DialogTitle>Bot Debug Information</DialogTitle>
+          <DialogDescription className="flex items-center gap-2">
+            Bot ID: {bot_uuid}
+            <button
+              onClick={handleViewGrafanaLogs}
+              className="text-xs text-primary hover:underline"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)